index.js

```diff
@@ -1,6 +1,7 @@
-import server from './src/server.js'
+import server from './src/server.js';
+import { envConfig } from './src/configs/env.js';
 
-server.listen(4000, () => {
+server.listen(envConfig.port, () => {
```

Comment:
By creating a file named `env.js` to configure and validate environment variables, we will replace the traditional method of using `process.env.ENV_VARIABLE_NAME` with `envConfig.ENV_VARIABLE_NAME`. This is done by importing { envConfig } from './src/configs/env.js'.